content/en/docs/setup/production-environment/turnkey/tencent.md

```diff
@@ -0,0 +1,26 @@
+---
+
+reviewers:
+- sftim
```

Comment:
Hi @yunxji 
I don't know who you should list here. I recommend _removing_ `reviewers:`

Reply:
OK, I did it.